src/invidious/views/components/player.ecr

```diff
@@ -15,6 +15,8 @@
                 <source src="/api/manifest/dash/id/<%= video.id %>?local=true&unique_res=1" type='application/dash+xml' label="dash">
             <% end %>
 
+            <% fmt_stream.reject! { |f| f["itag"] == 17 } %>
+            <% fmt_stream.sort_by! {|f| params.quality == f["quality"] ? 0 : 1 } %>
             <%
```

Comment:
```suggestion
            <%
              fmt_stream.reject! { |f| f["itag"] == 17 }
              fmt_stream.sort_by! {|f| params.quality == f["quality"] ? 0 : 1 }
```

Reply:
No need to add extra `<% %>`, there is already a chunk of Crystal code right below!

Also, I'm not sure: what is the purpose of `params.quality == f["quality"] ? 0 : 1` ?

Reply:
> Also, I'm not sure: what is the purpose of `params.quality == f["quality"] ? 0 : 1` ?

It's to have the selected/preferred video quality at the top of the video streams URLs because Firefox and Chrome only load the first video stream URL.

It's for non js users.

Reply:
Oh great! I was thinking about fixing that, but thanks for doing it!